rosbag2_cpp/src/rosbag2_cpp/writers/sequential_writer.cpp

```diff
@@ -79,6 +79,9 @@ void SequentialWriter::open(
 {
   max_bagfile_size_ = storage_options.max_bagfile_size;
   base_folder_ = storage_options.uri;
+  chunk_size_ = storage_options.chunk_size;
```

Comment:
I think this function should throw if `chunk_size_ == 0` is invalid.
Something like:
```c++
rcpputils::require_true(storage_options.chunk_size >= 1, "chunk_size must be at least 1.");
```
And then add `\throw std::invalid_argument` to the header documentation.

**This suggestion only applies if a value of 0 is invalid.**

Reply:
This is totally up for discussion. I think 0 is a valid entry as well. Maybe even a better indicator to disable the cache. Setting a cache size of 1 could then technically still use the internal cache.

Thoughts? Defaulting to zero and using the cache with everything except 0?

Reply:
Ok, so I've opted for the one I've described. Only setting it to `0` will disable the cache. Every other value will use the cache.